mobile/src/board/components/Tile.tsx

```diff
@@ -0,0 +1,90 @@
+import type { ReactNode } from "react";
+
+import { type BoardTile, Direction, GameState } from "../BoardTypes";
+
+import "./Tile.css";
+
+interface TreasureMap {
+  [key: string]: string;
+}
+
+export const treasures: TreasureMap = {
```

Comment:
I would rather uppercase constants when they : 
- are exported
- and will never change
( https://github.com/airbnb/javascript/#naming--uppercase ) 

Also, why not having used understandable keys? like heart for the heart...  it would be easier to read and prevent from making mistakes

Reply:
I did not used understandable keys as the treasure names inherits from the domain API